osu.Game/Screens/OnlinePlay/Multiplayer/Spectate/PlayerArea.cs

```diff
@@ -93,7 +98,13 @@ public void LoadScore([NotNull] Score score)
                 }
             };
 
-            stack.Push(new MultiSpectatorPlayerLoader(Score, () => new MultiSpectatorPlayer(Score, GameplayClock)));
+            stack.Push(new MultiSpectatorPlayerLoader(Score, () =>
+            {
+                var player = new MultiSpectatorPlayer(Score, GameplayClock);
+                player.OnGameplayStarted += () => OnGameplayStarted?.Invoke();
```

Comment:
I've changed this from `player.OnGameplayStarted += OnGameplayStarted` to what it is now. Do note that the former is not always correct because a delegate is null until it's bound to, so it basically becomes `player.OnGameplayStarted += null`.

Here's a test illustrating this:
```csharp
var c = new MyClass();
c.SecondAction += () => Console.WriteLine("AAA");
c.Call();

public class MyClass
{
    public event Action FirstAction;
    public event Action SecondAction;

    public MyClass()
    {
        FirstAction += SecondAction;
    }

    public void Call() => FirstAction();
}

Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at MyClass.Call() in /home/smgi/Workbench/EventTest/EventTest/Program.cs:line 17
   at Program.<Main>$(String[] args) in /home/smgi/Workbench/EventTest/EventTest/Program.cs:line 5

```

Reply:
Good catch, will keep that in mind next time.